index.html

```diff
@@ -0,0 +1,257 @@
+                    fill="hsl( var(--clr-silver) )"
+                  />
+                </svg>
+              </span>
+            </label>
+          </div>
+          <p class="fs-500 uppercase text-silver">Remember: X goes first</p>
+        </div>
+        <button
+          id="cpuBtn"
+          class="fs-700 text-dark uppercase d-block cpuBtn"
+          onclick="newGameCPU()"
+          data-value=""
+        >
+          New Game (vs CPU)
+        </button>
+        <button
+          id="playerBtn"
+          class="fs-700 text-dark uppercase d-block playerBtn"
+          onclick="newGamePlayer()"
+          data-value=""
+        >
+          New Game (vs player)
+        </button>
+      </section>
+
+      <section id="gameBoard" class="gameBoard">
+        <div class="game-board-header flex">
+          <img
+            class="game-board-header-logo"
+            src="./assets/images/logo.svg"
+            alt="logo"
+          />
+          <div class="turn">
+            <img
+              id="turn"
+              class="Xturn"
+              src="./assets/images/icon-x-outline.svg"
+              alt="X icon"
+              data-value="X"
+            />
+            <p class="fs-500 uppercase">turn</p>
+          </div>
+          <button class="restart" onclick="displayModalRestart()">
+            <img src="./assets/images/icon-restart.svg" alt="restart icon" />
+          </button>
+        </div>
+        <div class="boxes">
+          <div
+            onclick="hitBox('0')"
+            id="0"
+            class="box box0 hoverClassX"
+            data-value=""
+          ></div>
+          <div
+            onclick="hitBox('1')"
+            id="1"
+            class="box box1 hoverClassX"
+            data-value=""
+          ></div>
+          <div
+            onclick="hitBox('2')"
+            id="2"
+            class="box box2 hoverClassX"
+            data-value=""
+          ></div>
+          <div
+            onclick="hitBox('3')"
+            id="3"
+            class="box box3 hoverClassX"
+            data-value=""
+          ></div>
+          <div
+            onclick="hitBox('4')"
+            id="4"
+            class="box box4 hoverClassX"
+            data-value=""
+          ></div>
+          <div
+            onclick="hitBox('5')"
+            id="5"
+            class="box box5 hoverClassX"
+            data-value=""
+          ></div>
+          <div
+            onclick="hitBox('6')"
+            id="6"
+            class="box box6 hoverClassX"
+            data-value=""
+          ></div>
+          <div
+            onclick="hitBox('7')"
+            id="7"
+            class="box box7 hoverClassX"
+            data-value=""
+          ></div>
+          <div
+            onclick="hitBox('8')"
+            id="8"
+            class="box box8 hoverClassX"
+            data-value=""
```

Comment:
Move click handlers to a script file. Use event delegation to register these events instead of adding a click handler to all the elements here. https://javascript.info/event-delegation